app/build.gradle

```diff
@@ -33,7 +33,9 @@ android {
     buildFeatures {
         viewBinding = true
     }
-
+    androidExtensions {
```

Comment:
This will only let the `parcelize` feature active from the `kotlin-android-extensions` plugin, basically deactivating the synthetic views one. We might use this feature to create `Parcelable` classes just with an annotation :D

Reply:
I mean, it's cool. But if we're gonna introduce the dependency shouldn't we use `@Parcelize` on our `Parcelable` classes while we're at it?

Reply:
OK far more work than I thought :/ Think it's an improvement overall though. Thanks!